PWGHF/TableProducer/candidateSelectorB0ToDPi.cxx

```diff
@@ -67,16 +55,14 @@ struct HfCandidateSelectorB0ToDPi {
   Configurable<bool> activateQA{"activateQA", false, "Flag to enable QA histogram"};
   // check if selectionFlagD (defined in candidateCreatorB0.cxx) and usePid configurables are in sync
   bool selectionFlagDAndUsePidInSync = true;
-  // FIXME: store B0 creator configurable (until https://alice.its.cern.ch/jira/browse/O2-3582 solved)
-  int mySelectionFlagD = -1;
 
   TrackSelectorPID selectorPion;
 
   using TracksPIDWithSel = soa::Join<aod::BigTracksPIDExtended, aod::TrackSelection>;
 
   HistogramRegistry registry{"registry"};
 
-  void init(InitContext const& initContext)
+  void init(InitContext& initContext)
```

Comment:
Why is the `const` removed?

Reply:
It did not compile with `const`.

Reply:
I get this kind of message:
![image](https://github.com/AliceO2Group/O2Physics/assets/77975111/9188c4aa-4d39-42e0-9d95-c6a5575cfec5)